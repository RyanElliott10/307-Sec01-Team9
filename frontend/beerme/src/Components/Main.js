import React from "react";
import { Switch, Route } from "react-router-dom";

import Explore from "./Explore";
import AccountEntry from "./AccountEntry";
import Home from "./Home";

const Main = () => {
  return (
    <Switch>
      <Route exact path="/" component={Home} />
      <Route exact path="/explore" component={Explore} />
      <Route exact path="/account-entry" component={AccountEntry} />
    </Switch>
  );
};

export default Main;
