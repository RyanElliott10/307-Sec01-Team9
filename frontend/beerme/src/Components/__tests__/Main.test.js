import React from "react";
import renderer from "react-test-renderer";
import { MemoryRouter as Router } from "react-router-dom";

import Main from "../Main";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <Router>
        <Main />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
