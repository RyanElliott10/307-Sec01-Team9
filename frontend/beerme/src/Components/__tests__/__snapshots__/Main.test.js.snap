// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    style={
      Object {
        "alignContent": "center",
        "marginLeft": "200px",
        "marginRight": "200px",
        "marginTop": "45px",
      }
    }
  >
    <div
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "flexDirection": "column",
        }
      }
    >
      <img
        alt="Logo"
        src="BeerMe_Logo.png"
      />
    </div>
    <div
      style={
        Object {
          "marginBottom": "40px",
          "marginLeft": "auto",
          "marginRight": "auto",
          "width": "50%",
        }
      }
    >
      <div
        className="sc-htpNat dePyzG"
      >
        <input
          className="sc-bdVaJa jwfbbd"
          onChange={[Function]}
          placeholder="Search"
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</div>
`;
